### DISCUSSION TITLE
Bug in document names

### DISCUSSION THREAD

**Discussion by BorisFromBE** [openkfw/TruBudget#1146] [General]
It is currently impossible to upload documents using letters with accents to a workflow item, the error message given when trying to is also inexplicit, stating « You are not authorized to perform all listed actions ». 

We aim to allow the application to handle document names with accents.

**Comment by Stezido** [comment]
This issue can be related to #1054.
Even if the bug is fixed, we should change the error message so I transform this discussion into an issue.
